From 6aae821ae16b3166d6fdec4a4818d14aa0c0c5da Mon Sep 17 00:00:00 2001
From: Alin Jerpelea <alin.jerpelea@sony.com>
Date: Sat, 26 May 2018 09:03:55 +0200
Subject: [PATCH] display: remove blob dependency

there should not be any blob dependency at build time

Test: manual - build is ok

Change-Id: Id0b2887dec0ebc6832ee48767444c01a85ca34f8
Signed-off-by: Alin Jerpelea <alin.jerpelea@sony.com>
---
 sdm845/liblight/Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sdm845/liblight/Android.mk b/sdm845/liblight/Android.mk
index 2f2e1cac..71a4071a 100644
--- a/sdm845/liblight/Android.mk
+++ b/sdm845/liblight/Android.mk
@@ -22,7 +22,7 @@ LOCAL_C_INCLUDES += $(TARGET_OUT_HEADERS)/qdcm/inc
 
 LOCAL_SRC_FILES := lights.c lights_prv.cpp
 LOCAL_MODULE_RELATIVE_PATH := hw
-LOCAL_SHARED_LIBRARIES := liblog libcutils libsdm-disp-vndapis
+LOCAL_SHARED_LIBRARIES := liblog libcutils
 LOCAL_CFLAGS := -DLOG_TAG=\"qdlights\"
 LOCAL_CLANG  := true
 LOCAL_MODULE := lights.$(TARGET_BOARD_PLATFORM)
-- 
2.15.1

